website/www/site/content/en/documentation/ml/preprocess-data.md

```diff
@@ -70,9 +70,13 @@ TensorFlow documentation.
 | TFIDF | See [`tft.tfidf`](https://www.tensorflow.org/tfx/transform/api_docs/python/tft/tfidf) in the TensorFlow documentation. |:
 {{< /table >}}
 
-Apply the transforms on either single or multiple columns passed as a
-`dict` on structured data. Keys are column names and values are lists containing
-each column's data.
+### Generate Text Embeddings
+
```

Comment:
Add a sentence after the heading but before the table:

You can use `MLTranfrorm` to generate embeddings that you can use to push data into vector databases or to run inference.

Reply:
Done